python/ccf/clients.py

```diff
@@ -181,7 +181,7 @@ class Response:
 
     def __str__(self):
         versioned = (self.view, self.seqno) != (None, None)
-        status_color = "red" if self.status_code / 100 in (4, 5) else "green"
+        status_color = "red" if int(self.status_code / 100) in (4, 5) else "green"
```

Comment:
Is status code now a string?

Reply:
I think this is just
```suggestion
        status_color = "red" if self.status_code // 100 in (4, 5) else "green"
```

Reply:
This used to be a float division, which only worked for `400` and `500`. I didn't know about `//` which is neater!